pkg/ebpf/lockcontention.go

```diff
@@ -437,18 +442,32 @@ func (l *LockContentionCollector) Initialize(trackAllResources bool) error {
 		return int(int64(a.Start) - int64(b.Start))
 	})
 
-	keys := make([]uint32, ranges)
-	values := make([]LockRange, cpus*ranges)
-	var i, j uint32
-	for i = 0; i < ranges; i++ {
-		keys[i] = i
-		for j = 0; j < cpus; j++ {
-			values[(j*ranges)+i] = lockRanges[i]
-		}
+	batchSz := uint32(updateBatchSize)
```

Comment:
let's call it batchSize, I think the Hungarian notation isn't aligned with idiomatic go for naming